services/site/src/components/Footer.tsx

```diff
@@ -39,23 +39,6 @@ const Footer: React.FunctionComponent = () => (
       </nav>
```

Comment:
I think we can remove the nav, ul, and li here and just have the link. Otherwise, we would probably have to wrap the logo-link in the header in a nav also (for consistency). Not sure what the best option a11y wise is, but my guess is that a nav with just 1 element is confusing and unnecessary.